src/rocm_docs/rocm_docs_theme/flavors/instinct/header.jinja

```diff
@@ -6,7 +6,7 @@
     {% else %}
         {% set version_name = branch %}
     {% endif %}
-    <a class="klavika-font hover-opacity" href="{{ projects['rocm'] }}">Instinct&#8482; Hub</a>
+    <a class="klavika-font hover-opacity" href="{{ projects['rocm'] }}">Instinct&#8482; Documentation</a>
```

Comment:
Suggest removing TM symbol - shouldn't be in the Header

Reply:
TM is already in the ROCm docs header.
![image](https://github.com/user-attachments/assets/5417b0ca-de92-4fab-9131-9ded0110401f)